### ISSUE TITLE
ON1

### ISSUE THREAD

**Issue by fragandi** [TeamBasedInquiryLearning/linear-algebra#393]
Use the dot product to determine norms, distances, and angles.

**Comment by fragandi** [comment]
Current draft of problem mirroring [Activity 6.1.10](https://abb82c17.tbil.pages.dev/linear-algebra/preview/instructor/ON1#ON1-4-12) 
https://abb82c17.tbil.pages.dev/linear-algebra/preview/instructor/ON1#ON1-4-12

Given vectors u, v, w
1. Find the unit vectors
2. Which pairs of vectors are orthogonal?
3. Find the angle between the vectors that are not orthogonal

**Comment by jbunn3** [comment]
Thanks. As a heads up, I met with Manoj (@mlamich1) earlier, and he is going to help with this CheckIt problem as well. I'll let him chime in here.

**Comment by mlamich1** [comment]
I was out for lunch.  Just got here. I will be looking over it.

**Comment by mlamich1** [comment]
@jbunn3  Do you mean something similar 
Consider two vectors \vec{u} = (1, -3, 0, 4 ) and \vec{v} = ( 3, -3, 2, 1)  on R^4. 
Activity 1: Use dot product to determine the ||u|| and ||v||.
Activity 2: Using dot product, find the distance between \vec{u} = (1, -3, 0, 4 ) and \vec{v} = ( 3, -3, 2, 1)  on R^4.
Activity 3: Find the angle between \vec{u} and  \vec{v}.
or we need to make handout on for these problems since 6.1 does not talk about angle between two vectors.

**Comment by jbunn3** [comment]
@mlamich1 Yeah, but CheckIt will generate the particular vectors. See what @fragandi has above, which mimics a fluency building activity in the text. We just didn't want them to have to compute an entire orthogonal complement for the CheckIt problem. 

Also, keep in mind that 6.1 is incomplete; we still need to add the content about the law of cosines and how to find the angle between vectors.
